neurobooth_os/iout/lsl_streamer.py

```diff
@@ -61,7 +43,7 @@ def start_lsl_threads(node_name, collection_id="mvp_030", win=None, conn=None):
     for dc in kwarg_devs.values():
         kwarg_alldevs.update(dc)
 
-    scann_BLE()
+    # scan_BLE()
```

Comment:
delete?

Reply:
Yes; but I am following this up with a refactor of the mbient code. I'll aim to get rid of it then, I wanted to temporarily remember where it is for debugging purposes.